src/wp-includes/admin-bar.php

```diff
@@ -985,6 +985,17 @@ function wp_admin_bar_appearance_menu( $wp_admin_bar ) {
 		)
 	);
 
+	if ( current_user_can( 'activate_plugins' ) ) {
+		$wp_admin_bar->add_node(
+			array(
+				'parent' => 'site-name',
```

Comment:
Consistent with the other menu items, I think the 'parent` should be the `id` of the group - in this case 'appearance'.